infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -46,8 +47,9 @@
                     + "\tlog_replication_server (-l <path>|-m) [-nsN] [-a <address>|-q <interface-name>] "
                     + "[--max-num-snapshot-msg-per-batch=<batch-size>] "
                     + "[--max-data-message-size=<msg-size>] "
-                    + "[--lock-lease=<lease-duration>]"
                     + "[-c <ratio>] [-d <level>] [-p <seconds>] "
+                    + "[--node-id=<nodeId>]"
+                    + "[--log-replication-config=<log-replication-config-file-path>]"
```

Comment:
Whatever is added on the CorfuInterClusterReplicationServer we need to add in CorfuServer (as optional) because we start CorfuReplicationServer through the CorfuServer script (it's a license limitation that we have)